### ISSUE TITLE
Awesome workflows for alfred4

### ISSUE THREAD

**Issue by JiangWeixian** [JiangWeixian/cheatsheets#28]
> alfred@4 self-use awesome workflows

### 能不能好好说话

拼音缩写翻译。可以在[这里](https://lab.magiconch.com/nbnhhsh/)预览效果

<https://github.com/TheColdVoid/nbnhhsh>

### time zone

![tz](https://jhartman.pl/wp-content/uploads/2016/12/Alfred2.png)

<http://www.packal.org/workflow/timezones-2>

### caffeinate

就是那个控制亮屏时间的小工具

<http://www.packal.org/workflow/caffeinate-control>

### case

几种字符串类型的快速切换

![case](https://github.com/gillibrand/alfred-change-case/raw/master/changecase.png)

<https://github.com/gillibrand/alfred-change-case>

### toggle airpods

快速的连接airpods（不知道新的macos支不支持快速切换）

![airpods](https://github.com/jasonshanks/alfred-workflow-toggle-airpods/raw/master/images/setairpods.gif)

<https://github.com/jasonshanks/alfred-workflow-toggle-airpods>

### terminal finder

在finder中打开iterm，或者在iterm打开finder。和当前focus的finder or iterm窗口有关

![tf](https://camo.githubusercontent.com/e03d15b918738e254b309171014ae33a43da4057/68747470733a2f2f7261772e6769746875622e636f6d2f4c65456e6e6f2f616c667265642d7465726d696e616c66696e6465722f6d61737465722f73637265656e73686f745f66742e706e67)

<https://github.com/LeEnno/alfred-terminalfinder>

### wechat

use wechat in alfred

<https://github.com/MustangYM/WeChatExtension-ForMac>

### switch windows
switch windows like context

![switch](https://github.com/mandrigin/AlfredSwitchWindows/raw/master/demo_concise.gif)

<https://github.com/mandrigin/AlfredSwitchWindows>

### hash

encode with hash md5 etc..

![hash](https://github.com/willfarrell/alfred-hash-workflow/raw/master/screenshots/hash.png)

<https://github.com/willfarrell/alfred-hash-workflow>

### lorem

lorem words

![lorem](https://raw.githubusercontent.com/tillkruss/alfred-lorem-ipsum/master/screenshot.png)

<https://www.packal.org/workflow/lorem-ipsum-0>

### httpstat

show website info 

![httpstat](https://github.com/leozhang2018/alfred-httpstat/raw/master/screenshot.jpg)

<https://github.com/leozhang2018/alfred-httpstat>

### tldr

show cheatsheet of command

![tldr](https://github.com/cs1707/tldr-alfred/blob/master/screenshot.gif?raw=true)

<https://github.com/cs1707/tldr-alfred>

### emoji

search emoji by word and content

![emoji](https://github.com/sindresorhus/alfred-emoj/blob/master/screenshot-combined.png?raw=true)

<https://github.com/sindresorhus/alfred-emoj>

### colors

show color

![colros](https://cloud.githubusercontent.com/assets/398893/14360267/6d0aa34c-fcaa-11e5-9c35-fef24188842d.png)

<http://www.packal.org/workflow/colors>

### devdocs

devdocs search，搜索体验可能比不上dash，但是开源免费。

![devdocs](https://camo.githubusercontent.com/a85f1f1391e2616ed1556ec2f7ff00d8dd92e613/687474703a2f2f636f6e74656e742e73637265656e636173742e636f6d2f75736572732f79616e6e69636b676c742f666f6c646572732f536e616769742f6d656469612f65623934333231392d353237352d346366382d613931352d6139376561313737326661302f323031342d30382d30345f31342d34332d31332e706e67)

<https://github.com/yannickglt/alfred-devdocs>

### ip

show current ip info

![ip](https://cloud.githubusercontent.com/assets/398893/14361428/ab1e7910-fcaf-11e5-86e8-51f7a5f44aa8.png)

<https://github.com/zenorocha/alfred-workflows#ip-address-v120--download>

### vscode

open project in vscode

![image](https://user-images.githubusercontent.com/6839576/81713771-deb45c00-94a8-11ea-9c1c-44399a288599.png)

需要配置文件夹路径所在位置，可以通过设置变量方式实现 <https://juejin.im/post/5d880368e51d4561c41fb906>

### translate

show tranlate result in alfred

![translate](https://github.com/xfslove/alfred-google-translate/raw/master/media/corrected.png)

需要配合alfred-google-translate-config使用，国内请使用translate.cn，请关闭voice

设置`trc en&zh-CN`

复制通过`cmd c`

<https://github.com/xfslove/alfred-google-translate>

### kill-process

命令行的方式杀死软件

![kill-process](https://github.com/ngreenstein/alfred-process-killer/raw/master/screenshot1.png)

<https://cloud.githubusercontent.com/assets/398893/14360276/6d2a33ba-fcaa-11e5-8fa5-4d3703a8129f.png>

### pkg-management

search pkg in alfred，支持多个平台比如npm等。

![pkgman](https://github.com/willfarrell/alfred-pkgman-workflow/raw/master/screenshots/alcatraz.png)

<https://github.com/willfarrell/alfred-pkgman-workflow>

### base64 string

base64

![encode](https://github.com/willfarrell/alfred-encode-decode-workflow/raw/master/screenshots/encode.png)

<https://github.com/willfarrell/alfred-encode-decode-workflow/raw/master/screenshots/encode.png>

### qrcode

![qrcode](https://github.com/cj1128/alfred-qrcode-workflow/raw/master/intro.gif)

<https://github.com/willfarrell/alfred-encode-decode-workflow/raw/master/screenshots/encode.png>
